utilities/templates/htcondor/htboinc_run_six.sub

```diff
@@ -0,0 +1,15 @@
+# A.Mereghetti, 2017-03-07
```

Comment:
I guess that it is better if we deal with the path to the .zip file rather than to only the dir